utils/src/compress/lz4_standard.rs

```diff
@@ -15,20 +15,20 @@ pub(super) fn lz4_compress(src: &[u8]) -> Result<Vec<u8>> {
     }
 
     let mut dst_buf = Vec::with_capacity(compress_bound as usize);
-    let dec_size = unsafe {
+    let cmp_size = unsafe {
```

Comment:
These changes are irrelevant to the zstd support patch, can you please move it into a separate one?

Reply:
Ok